index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: index
+---
+
+## Key concepts
+
+> Auditree aims to be opinionated in how you work, but flexible in what work
+> you do.
+
+Everything revolves around *evidence*. Simple programs called *fetchers*
+retrieve evidence from sources (API's, URL's, command line invocations) and
+store that evidence in a *locker*. This is a git repository (so efficient,
+tamper evident and versioned for storing evidence) with some additional
+controls & metadata inserted by the supporting framework. Evidence can then be
+verified by *checks*; simple unit tests against your evidence.
+
+Your checks can be use to assert an operational posture, implement secondary
+controls or be a primary control in your compliance posture.
+
+Notification of check status can be sent to a issues, Slack, PagerDuty, or
+even stored alongside evidence. Playbooks for how to respond to check warnings
+or failures can be linked to from these notificiations.
+
+## The tools
+
+The Auditree system is designed to be extensible by those using it. It
+provides a framework in which to build & run fetchers & checks, defines an
+operational model for approaching compliance activities and supplies ancillary
+tools to facilitate a workflow around that operational model.
+
+- [Auditree framework][framework] - the core of the system, responsible for managing evidence in the locker, running fetchers & checks, notification and reporting
+- [Arboretum][arboretum] - a library of open fetchers & checks for you to use & contribute to
+- [Harvest][harvest] - run reports & analysis over the contents of the locker
```

Comment:
Harvest also collates or gathers evidence.  I think it's worthwhile mentioning so that feature doesn't get obscured by the reporting capabilities.